template/gitlab-runner.tftpl

```diff
@@ -58,8 +58,16 @@ then
       --form "maximum_timeout=${gitlab_runner_maximum_timeout}" \
       --form "runner_type=${gitlab_runner_type}_type" \
       $runner_type_param \
-      --form "access_level=${gitlab_runner_access_level}" \
-      | jq -r '.token')
+      --form "access_level=${gitlab_runner_access_level}")
+
+    token = $(echo response | jq -r '.token')
+    if [[ "$token" == null ]]
+        message = $(echo response | jq -r '.message // .error_description')
+        if [[ "$message" != null ]]
+            echo "ERROR: API call returned $message".
```

Comment:
suggestion: Give some more information to the user which API call failed to make debugging easier.

```suggestion
            echo "ERROR: Couldn't register the Runner. GitLab API call returned $message".
```

Reply:
I have updated the code as suggested.